client/my-sites/domains/domain-management/edit/registered-domain.jsx

```diff
@@ -71,6 +72,12 @@ const RegisteredDomain = createReactClass( {
 		);
 	},
 
+	getGdprConsentMessage() {
+		const { domain, translate } = this.props;
+		const message = getGdprConsentStatusMessage( domain );
```

Comment:
One last thing - I would not show this row at all if there is no consent status returned from the API. That will also gracefully handle WWD domains.

Reply:
Yeah. Great point. Thanks.

Reply:
No need to be sorry. You're right, of course.  ...will update. :-)

Reply:
Kinda like this pattern more. I mean, bail early, and have easy to read code below.